cmd/protoc-gen-fastmarshal/templates/fieldsnippets.tmpl

```diff
@@ -855,6 +855,62 @@
             m.{{.GoName}} = {{if eq $syntax "proto2"}}csproto.Int{{$bitSize}}(v){{else}}v{{end}}
         }
     {{- end -}}
+{{- else if eq $kind "float" -}}
```

Comment:
Can you also add `float` and `double` fields to the `AllTheThings` and `RepeatAllTheThings` message definitions, along with the corresponding tests, in the various examples?